bin/doctrine-migrations.php

```diff
@@ -77,5 +77,6 @@
     ? include 'migrations-output.php'
```

Comment:
these input and output extension points are not supported anymore (so should be dropped)

Reply:
@stof removed here #686